packages/shared/styles/components/atoms/SfIcon.scss

```diff
@@ -11,6 +12,34 @@
     width: inherit;
     height: inherit;
   }
+  &__badge {
```

Comment:
@AdamPawlinski and @martaradziszewska what do you think about including here SfBadge and add modifier --rounded or --is-rounded now we duplicate code for it here and in SfCircleIcon

Reply:
@AdamPawlinski @pspaczek  good idea, I totally agree. :)

Reply:
OK I'll change that :)

Reply:
I added --rounded modifier to badge and story reflecting it, in SfIcon I changed __badge to SfBadge and fixed showing badge in Header task https://github.com/DivanteLtd/storefront-ui/issues/1061 (it showed even if quantity was 0)